lib/arturo/feature_caching.rb

```diff
@@ -36,12 +36,13 @@ def to_feature(feature_or_symbol)
     def self.extended(base)
       class << base
         prepend PrependMethods
-        attr_accessor :cache_ttl, :feature_cache, :feature_caching_strategy
+        attr_accessor :cache_ttl, :feature_cache, :feature_caching_strategy, :extend_cache_on_failure
```

Comment:
I think I would prefer only a writer method for `:extend_cache_on_failure`, and then add a `#extend_cache_on_failure?` method as the reader, e.g.
```rb
def extend_cache_on_failure?
  !!@extend_cache_on_failure
end
```

Reply:
good idea. Done